app/lib/MyAccountsStore.js

```diff
@@ -101,8 +120,9 @@ export default class MyAccountsStore {
             : konnectors.install(cozy.client, konnector.slug, konnector.repo, INSTALL_TIMEOUT * 1000)
 
         return installationPromise
-          .then(konnector => {
-            return konnectors.addAccount(cozy.client, konnector, account)
+          .then(konnectorResult => {
+            konnector.links = konnectorResult.links
```

Comment:
Should we call `this.updateConnector(konnector)` here ? And deal with the `link` property inside this method ?